README.md

```diff
@@ -1 +1,150 @@
+
+The baseline includes tests that must be executed against the Kubernetes API and tests that must be executed against the underlying nodes over SSH. This profile handles only the node tests, and is intended to be run alongside the [CMS ARS 5.0 EKS CIS Cluster Overlay profile](https://github.com/CMS-Enterprise/cms-ars-5.0-eks-cis-cluster-overlay).
+
+## Getting Started  
+### InSpec (CINC-auditor) setup
+For maximum flexibility/accessibility, we’re moving to “cinc-auditor”, the open-source packaged binary version of Chef InSpec, compiled by the CINC (CINC Is Not Chef) project in coordination with Chef using Chef’s always-open-source InSpec source code. For more information: https://cinc.sh/
+
+It is intended and recommended that CINC-auditor and this profile overlay be run from a __"runner"__ host (such as a DevOps orchestration server, an administrative management system, or a developer's workstation/laptop) against the target. This can be any Unix/Linux/MacOS or Windows runner host, with access to the Internet.
+
+__For the best security of the runner, always install on the runner the _latest version_ of CINC-auditor.__ 
+
+__The simplest way to install CINC-auditor is to use this command for a UNIX/Linux/MacOS runner platform:__
+```
+curl -L https://omnitruck.cinc.sh/install.sh | sudo bash -s -- -P cinc-auditor
+```
+
+__or this command for Windows runner platform (Powershell):__
+```
+. { iwr -useb https://omnitruck.cinc.sh/install.ps1 } | iex; install -project cinc-auditor
+```
+To confirm successful install of cinc-auditor:
+```
+cinc-auditor -v
+```
+> sample output:  _4.24.32_
+
+Latest versions and other installation options are available at https://cinc.sh/start/auditor/.
+
+## Specify your BASELINE system categorization as an environment variable
+### (if undefined defaults to Moderate baseline)
+
+```
+# BASELINE (choices: Low, Low-HVA, Moderate, Moderate-HVA, High, High-HVA)
+# (if undefined defaults to Moderate baseline)
+
+on linux:
+BASELINE=High
+
+on Powershell:
+$env:BASELINE="High"
+```
+## Tailoring to Your Environment
+
+The following inputs may be configured in an inputs ".yml" file for the profile to run correctly for your specific environment. More information about InSpec inputs can be found in the [InSpec Profile Documentation](https://www.inspec.io/docs/reference/profiles/).
+
+```yaml
+# Used by check 3.2.3. Set this value to a string containing the full path
+# ("/example/of/path/string/for/ca/file.crt")
+# to the certificate authority file used for kubelet to authenticate certificates.
+client_ca_file_path:
+
+# Used by checks 3.2.10 and 3.2.11. Set this to `false` if your node kubelet processes
+# get certificates from the k8s API server, or `true` if your certificates
+# are handled by a process outside of k8s.
+# Acceptable values (boolean): true false
+external_cert_authority_in_use:
+```
+
+## Running the Profile
```

Comment:
Question on this section:
Do the `running the profile directly` commands not need `sudo` and do the `running from an archive` commands not need `BASELINE=<sys_cat>`? I could be missing some context here.

Reply:
I added the baseline statements, thanks.
An  ec2-user seemed sufficient without additional privilege.